blazor/mention/working-with-data.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: post
+title: Working with data in Blazor Mention Component | Syncfusion
+description: Checkout and learn here all about working with data in Syncfusion Blazor Mention component and more.
+platform: Blazor
+control: Mention
+documentation: ug
+---
+
+# Working with Data in Blazor Mention Component
+
+The Mention allows you to bind data either from a local source or a remote data service using the `DataSource` property. If you want to bind data from a local source, you can simply assign an enumerable list of data items to the `DataSource` property. If you want to bind data from a remote data service, you can use the [DataManager](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DataManager.html) component to fetch and bind the data to the `Mention` component.
+
+* **TItem** - Specifies the type of the `DataSource` of the Mention component.
+
+## Binding local data
+
+The Mention component typically loads its data from a local data source, such as an array of strings or an array of objects.
+
+The `DataSource` property of the Mention component specifies the data that will be used to populate the list of options that users can choose from when mentioning the item. The `DataSource` property supports several different data types, including:
+
+* `Array of primitive type`: This is an array of simple data types, such as strings or numbers.
```

Comment:
* `Array of primitive type`: This is an array of simple data types, such as strings or numbers.
* `List of primitive type`: This is a list (a type of collection) of simple data types, such as strings or numbers.
* `IEnumerable<TItem>`: This is a sequence of items that can be enumerated.
* `ExpandoObject`: This is a type of object that allows new properties to be added dynamically at runtime.

Reply:
Addressed